content/sales.rst

```diff
@@ -0,0 +1,196 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describe operational Sales details.
+
+The main contact is Tim Adams, the main email group is at sales@proatria.com.
+
+Sales:
+
+* Provides the main software information to prospective customers, forwards
+  additional technical support queries to Support.
+* Introductory email for the documentation, installation guides and downloads
+  page.
+* Pricing information, including specific pricing for the general and
+  additional/extra services.
+* Licensing information, contracts.
+
+
+Replying to Sales Prospects
+===========================
+
+Sales message templates are used so that prospects are given consistent
+information.
+
+Please ask the Sales team for these templates, or forward the query to
+sales@proatria.com.
```

Comment:
We should put the templates into WIKI or into the support system.

but for the styleguide, we can try to give some examples of these templates